lib/gettext-setup/pot.rb

```diff
@@ -50,9 +50,14 @@ def self.string_changes?(old_pot, new_pot)
       return true
     end
 
-    def self.generate_new_pot(locales_path: GettextSetup.locales_path, target_path: nil)
+    # @param [:locales_path] opts
+    #   The directory for the locales.
+    # @param [:target_path] opts
+    #   The output path for the new POT file.
+    def self.generate_new_pot(opts = {})
```

Comment:
will these functions poop if no hash is passed in?

Reply:
No, because the `= {}` passes an empty hash by default. So `[]` will just return nil, which we handle.

Reply:
![image](https://user-images.githubusercontent.com/7181606/29987538-90663f2a-8f1d-11e7-92bc-b49a1b02b577.png)

Reply:
They should pass, filling in defaults below. Unless `nil` is passed in explicitly.